public/js/test/integration/todomvc.js

```diff
@@ -14,6 +14,11 @@ function goToSource(source) {
 }
 
 function togglBreakpoint(linenumber) {
+  cy.window().then(win => {
+    cy.wrap(win.cm)
+      .invoke("scrollIntoView", { line: linenumber, ch: 0 });
+  });
```

Comment:
This command scrolls codemirror to the breakpoint line we want to toggle.

This is the only place where codemirror is used in the tests and I figured it was worth exposing codemirror so the tests would not have to figureout where the line was.

`cy.window`, `cy.wrap`, `invoke` are somewhat awkward functions for getting the debugger window context, and invoking functions on window objects. They're documented here: https://docs.cypress.io/v1.0/docs/invoke and simpler than the mochitest equivelants https://docs.cypress.io/v1.0/docs/invoke.